### DISCUSSION TITLE
ArrayTable在@formily/vue 2.1.0版本中无法正常使用

### DISCUSSION THREAD

**Discussion by wangwangtou** [alibaba/formily#3097] [Q&A 2.x] [answered]
@formily/element 官方ArrayTable例子，在新版本下无法正常运行

formily2.1.0
https://codesandbox.io/s/goofy-hertz-thmiex?file=/src/App.vue
添加按钮无效

formily2.0.11
https://codesandbox.io/s/thirsty-bassi-2ny8br?file=/src/App.vue
正常运行

**Comment by janryWang** [comment, marked as answer]
重复了 https://github.com/alibaba/formily/issues/3093
